artifactory/services/remoterepository.go

```diff
@@ -193,6 +193,7 @@ type RemoteRepositoryBaseParams struct {
 	ClientTlsCertificate              string                  `json:"clientTlsCertificate,omitempty"`
 	ContentSynchronisation            *ContentSynchronisation `json:"contentSynchronisation,omitempty"`
 	QueryParams                       string                  `json:"queryParams,omitempty"`
+	CdnRedirect                       *bool                   `json:"cdnRedirect,omitempty"`
```

Comment:
Looks like RemoteRepositoryBaseParams is already inherit AdditionalRepositoryBaseParams
```suggestion
```